neuro_config_client/factories.py

```diff
@@ -814,3 +815,34 @@ def create_ingress(cls, ingress: IngressConfig) -> dict[str, Any]:
         if ingress.cors_origins:
             result["cors_origins"] = ingress.cors_origins
         return result
+
+    @classmethod
+    def create_node_pool(cls, node_pool: NodePool) -> dict[str, Any]:
```

Comment:
This payload cannot be used for adding/updating node pools. Payload depends on the type of cluster cloud provider: for cloud clusters machine configuration is taken from node templates which are stored on the server, for onprem clusters machine configuration is provided by client

Reply:
Good catch, thank you for help! checkout [be58386](https://github.com/neuro-inc/neuro-config-client/pull/83/commits/be5838666c221fd64f7f1085e93d2598905e231a)